src/DynamoCore/Search/SearchElements/NodeModelSearchElement.cs

```diff
@@ -12,19 +12,25 @@ namespace Dynamo.Search.SearchElements
     public class NodeModelSearchElement : NodeModelSearchElementBase
     {
         private readonly Func<NodeModel> constructor;
+        private readonly string creationName;
 
+        public override string CreationName { get {return creationName;} }
+        
```

Comment:
one thought, when we do the JSON serialization of nodes, we have a field called `concreteType` that looks like:

` "ConcreteType":"Dynamo.Graph.Nodes.ZeroTouch.DSFunction, DynamoCore",`
this is almost the assembly qualified name without the version number and culture etc...

we're using this in various places, it might be nice if the creationName was similar to this concreteType, what do you think, is just the typeName better?